server/cmwell-cons/docker.readme

```diff
@@ -0,0 +1,20 @@
+To build a docker image:
+
+1: Use sbt and run packageCmwell
+2: Execute the following command in this directory:
```

Comment:
`in this directory` you mean from server/cmwell-cons as I see that u placed the docker file at this location.?